backend/core/src/applications/services/applications.service.ts

```diff
@@ -324,7 +324,7 @@ export class ApplicationsService {
           try {
             application = await this._createApplication(applicationCreateDto)
           } catch (e) {
-            console.error(e.message)
+            console.error(`${e.message} - listing id: ${applicationCreateDto.listingId}`)
```

Comment:
Not sure if this is where we would need to log to get the listing id on app submission errors